### DISCUSSION TITLE
无法启动服务端

### DISCUSSION THREAD

**Discussion by Kwaiyu** [UnblockNeteaseMusic/install-scripts#19] [Q&A]
求大佬帮忙看下，都是默认设定。
系统是：CentOS Linux release 7.9.2009 (Core)
![image](https://user-images.githubusercontent.com/28617728/224099504-d77834ee-d682-4d91-9e5b-59f489857215.png)

![image](https://user-images.githubusercontent.com/28617728/224098820-9f0311bf-1d96-4416-9102-bfb25d89d42d.png)

将 is-active 全部替换为 lsof -i:7999 还是无法启动服务端
![image](https://user-images.githubusercontent.com/28617728/224099842-3cde58b6-f38d-47e2-bf9a-5eef90ed2966.png)

![image](https://user-images.githubusercontent.com/28617728/224100152-d1aa7cfa-1da1-4dac-a7f1-ce2eeb277930.png)

**Comment by 1715173329** [comment]
node -v 看看，可能 centos7 太老了带不动了。
